modules/simple-payments/paypal-express-checkout.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * This PaypalExpressCheckout global is included by wp_enqueue_script( 'paypal-express-checkout' );
+ * It handles communication with Paypal Express checkout and public-api.wordpress.com for the purposes
+ * of simple-payments module.
+ */
+
+/* global paypal */
+/* exported PaypalExpressCheckout */
+/* jshint unused:false */
+var PaypalExpressCheckout = {
+	constants: {
+		createPaymentEndpoint: '', //TODO: point to the actual endpoint
+		executePaymentEndpoint: '' //TODO: point to the actual endpoint
+	},
+	renderButton: function( id ) {
```

Comment:
Just for defensive coding purposes (like if somehow this got included without the `paypal` global), do you think it would be helpful to add `if ( ! paypal ) { throw new Error( 'paypal module is required by PaypalExpressCheckout' ) }` or `if ( ! paypal ) { console.error( 'paypal module is required by PaypalExpressCheckout' ) && return }`?